File: VetCare/Models/Appointment.cs
namespace VetCare.Models
{
    using System;
    using System.Collections.Generic;
    using System.ComponentModel.DataAnnotations;
    using System.ComponentModel.DataAnnotations.Schema;
    using System.Data.Entity.Spatial;

    [Table("Appointment")]
    public partial class Appointment //dbye setlenen
    {
        public int ID { get; set; }

        [Required(ErrorMessage="Name is required!")]
        [StringLength(200)]
        public string Name { get; set; }

        [Required(ErrorMessage = "Phone is required!")]
        [StringLength(200)]
        public string Phone { get; set; }

        [Required(ErrorMessage = "Email is required!")]
        [StringLength(200)]
        public string Email { get; set; }

        [Required(ErrorMessage = "Date is required!")]
        [Column(TypeName = "date")]
        public DateTime Date { get; set; }

        [Required(ErrorMessage = "Gender is required!")]
        [StringLength(200)]
        public string Gender { get; set; }

        [Required(ErrorMessage = "Reason is required!")]
        [StringLength(200)]
        public string Reason { get; set; }

        [StringLength(200)]
        public string AddInfo { get; set; }
    }
}

File: VetCare/Scripts/Appointment.js
function buttonFunction() {
    console.log("HHEEEYY!!!!");
    if (document.Appointment.Name == "") {
        alert("Please provide your name");
        document.Appointment.Name.focus();
        return false;
    }
    if (document.Appointment.Phone == "") {
        alert("Please provide your phone");
        document.Appointment.Phone.focus();
        return false;
    }
    if (document.Appointment.Email == "") {
        alert("Please provide your email");
        document.Appointment.Email.focus();
        return false;
    }
    if (document.Appointment.Date == "") {
        alert("Please provide your date");
        document.Appointment.Date.focus();
        return false;
    }
    if (document.Appointment.Gender == "") {
        alert("Please provide your gender");
        document.Appointment.Gender.focus();
        return false;
    }
    if (document.Appointment.Reason == "") {
        alert("Please provide your reason");
        document.Appointment.Reason.focus();
        return false;
    }

}